wallet/res/values/strings-extra.xml

```diff
@@ -284,16 +284,21 @@
     <string name="tools_show_xpub">Extended Public Key</string>
 
     <string name="about_help_us_improve">Help us improve your experience</string>
-    <string name="about_review_and_rate">Review &amp; Rate the app</string>
+    <string name="about_review_and_rate">Review and rate the app</string>
```

Comment:
this change will make it easier for translators.  The `&` causes some trouble and we have to manually fix some of the text that includes this symbol.